atomic_reactor/core.py

```diff
@@ -620,3 +621,32 @@ def get_version(self):
         :return: dict, json output of `docker version`
         """
         return self.d.version()
+
+    def get_volumes_for_container(self, container_id):
+        """
+        get a list of volumes mounter in a container
+
+        :param container_id: str
+        :return: list, a list of volume ids
+        """
+        logger.info("listing volumes for container '%s'", container_id)
+        inspect_output = self.d.inspect_container(container_id)
+        volumes = inspect_output['Mounts'] or []
+
+        volume_names = [x['Name'] for x in volumes if x['Source'] != ""]
+        logger.debug("volumes = '%s'", volume_names)
```

Comment:
Ah, right, I was thinking this would help to show an empty list, but its not really required. Fixed